packages/gatsby-cli/README.md

```diff
@@ -1,4 +1,5 @@
 # gatsby-cli
+[![npm version](https://badge.fury.io/js/gatsby-cli.svg)](https://badge.fury.io/js/gatsby-cli)
```

Comment:
We're not super fond of adding badges to our readme files as there are a lot of dependencies that we could add badges for, but I don't think that helps people in deciding to use Gatsby or not.

```suggestion
```

Reply:
Sure, applied the suggestion!